docs/github-on-chain-identity.md

```diff
@@ -0,0 +1,7 @@
+# Adding github account to your on-chain identity
+
+Adding the github account to your identity requires the usage of the `additional` fields in the identity info. This is currently only supported by using the [polkadot-js bare extrinsic interface](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/extrinsics). See the following image for an example:
```

Comment:
Might be worth emphasizing that the identities are set in the relay chain, while the fellowship itself lives in collectives?